src/screens/SpeedUpAndCancelSheet.js

```diff
@@ -132,9 +132,11 @@ export default function SpeedUpAndCancelSheet() {
   const [minGasPrice, setMinGasPrice] = useState(
     calcMinGasPriceAllowed(tx.gasPrice)
   );
+  const fetchedTx = useRef(false);
   const [gasLimit, setGasLimit] = useState(tx.gasLimit);
   const [data, setData] = useState(tx.data);
   const [value, setValue] = useState(tx.value);
+  const [nonce, setNonce] = useState(tx.nonce);
```

Comment:
Given that gasLimit, data, value, nonce are mainly set together and used together, should we have one useState "txPayload" or something instead of having these all separate?

Reply:
I prefer to keep it as it is so I can "pre-populate" with the values from tx. Otherwise I have to add a bunch conditionals after.